run_scans.py

```diff
@@ -104,8 +105,19 @@ def launchTestsArgs(tool, slot, link, chamber, scanmin, scanmax, nevts,
     cmd.append( "--scanmin=%d"%(scanmin) )
     cmd.append( "--scanmax=%d"%(scanmax) )
     cmd.append( "--nevts=%d"%(nevts) )
+    cmd.append( "--throttle=%i"%(throttle) )
```

Comment:
The default value that is passed is 0, and the default value in the options is 0.  I understood based on the implementation a value of 0 means no throttle.

So there should be no problem here. No?